### PR TITLE
add Re-claim Possessions coords

### PR DESCRIPTION
**Pull request by evanillaa**

### CODE REVIEW

**Comment by zfbx** [comment]
What is the purpose of this? you get your items back when you leave jail do you not?

**Comment by evanillaa** [comment]
Items will be returned after reaching the item collection area
and not automatically
Increased realism

File: config.lua [hunk 1]
```diff
@@ -45,6 +45,9 @@ Config.Locations = {
     ["shop"] = {
         coords = vector4(1786.19, 2557.77, 45.62, 0.5),
     },
+    ["reclaim"] = {
+        coords = vector4(1837.94, 2593.11, 45.89, 76.94),
```
Comment (phumberdroz) [inline config.lua:49]: Why make this even a vector4? a vector3 would make your code above much cleaner.

**Review by phumberdroz — COMMENTED** [review]
Generally I like it as it would make it so players have to collect their possessions after being released.

File: config.lua (hunk 1 above)
Comment (evanillaa) [reply config.lua:49]: I'm sorry, it looks like it was a mistake

Comment (evanillaa) [reply config.lua:49]: Can you explain to me what the goal is when you press resovle conversation 
I did not understand the purpose of it

**Pull request closed without merging**